### PR TITLE
69607: Add note about common rclone sync with dropbox error.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/tasks.rst [hunk 1]
```diff
@@ -168,11 +168,18 @@ shows the configuration options for Cloud Syncs.
    | Enabled                | checkbox            | Unset to temporarily disable this Cloud Sync.                                                          |
    |                        |                     |                                                                                                        |
    +------------------------+---------------------+--------------------------------------------------------------------------------------------------------+
-   
-   
+
+
 .. note:: The time selected is when the Cloud Sync task is allowed to
```
Comment (Mrt134) [inline userguide/tasks.rst]:
```suggestion
The time selected is when the Cloud Sync task is allowed to
```

File: userguide/tasks.rst [hunk 2]
```diff
@@ -168,11 +168,18 @@ shows the configuration options for Cloud Syncs.
...
    begin. The cloud sync runs until finished, even after the time selected.
```
Comment (Mrt134) [inline userguide/tasks.rst]:
```suggestion
begin. The cloud sync runs until finished, even after the time selected.
```

File: userguide/tasks.rst [hunk 3]
```diff
@@ -168,11 +168,18 @@ shows the configuration options for Cloud Syncs.
...
 
+.. note:: If
+   `rclone sync <https://rclone.org/commands/rclone_sync/>`__
+   encounters any errors, files are not deleted in the destination.
+   This includes a common error when the Dropbox
+   `copy right detector <https://techcrunch.com/2014/03/30/how-dropbox-knows-when-youre-sharing-copyrighted-stuff-without-actually-looking-at-your-stuff/>`__
+   flags a file as copyrighted.
```
Comment (Mrt134) [inline userguide/tasks.rst]:
```suggestion
   identifies a copyrighted file.
```

File: userguide/tasks.rst [hunk 4]
```diff
@@ -168,11 +168,18 @@ shows the configuration options for Cloud Syncs.
...
-   
-   
+
+
 .. note:: The time selected is when the Cloud Sync task is allowed to
    begin. The cloud sync runs until finished, even after the time selected.
 
+.. note:: If
+   `rclone sync <https://rclone.org/commands/rclone_sync/>`__
+   encounters any errors, files are not deleted in the destination.
```
Comment (Mrt134) [inline userguide/tasks.rst]: > If `rclone sync <https://rclone.org/commands/rclone_sync/>`__ encounters any errors, files are not deleted in the destination.

This is a little unclear to me. Are the "files" that are not deleted the files that were sent previously as part of the `rclone sync`, or other files on the destination system such as those from a previously completed rclone command?

Comment (a-stjohn) [reply userguide/tasks.rst]: According to `rclone sync` docs: "Make source and dest identical, modifying destination only." What I have makes sense to me. Feel free to make a suggestion. I can't seem to think of a way to make it clearer.

Comment (Mrt134) [reply userguide/tasks.rst]: How about:
`
Files that have completed the sync process are not deleted from the destination if the rclone sync <link> is interrupted or encounters an error.
`
That feels a little more definitive.

File: userguide/tasks.rst [hunk 5]
```diff
@@ -168,10 +168,18 @@ shows the configuration options for Cloud Syncs.
...
+
+The time selected is when the Cloud Sync task is allowed to
+begin. The cloud sync runs until finished, even after the time selected.
+
+.. note:: Files that have completed the sync process are not deleted
+   from the destination if the
+   `rclone sync <https://rclone.org/commands/rclone_sync/>`__
+   is interrupted or encounters an error. This includes a common error
+   when the Dropbox
+   `copy right detector <https://techcrunch.com/2014/03/30/how-dropbox-knows-when-youre-sharing-copyrighted-stuff-without-actually-looking-at-your-stuff/>`__
```
Comment (wblock) [inline userguide/tasks.rst]: s/copy right/copyright/

**Pull request merged**